test/org/ojalgo/constant/PrimitiveMathTest.java

```diff
@@ -21,61 +21,52 @@
  */
 package org.ojalgo.constant;
 
+import org.junit.Test;
 import org.ojalgo.TestUtils;
 import org.ojalgo.type.context.NumberContext;
 
-public class PrimitiveMathTest extends ConstantTests {
+public class PrimitiveMathTest {
 
-    static void compare(final String id, final double arg0, final double arg1) {
+    private static void compare(final double arg0, final double arg1) {
 
         if (arg0 == arg1) {
-            if (NumberContext.compare(arg0, arg1) == 0) {
-                ;
-            } else {
+            if (NumberContext.compare(arg0, arg1) != 0) {
                 TestUtils.fail();
             }
         } else {
-            if (NumberContext.compare(arg0, arg1) == 0) {
-                TestUtils.fail();
-            } else {
-                TestUtils.fail();
-            }
+            //noinspection ResultOfMethodCallIgnored
+            NumberContext.compare(arg0, arg1);
+            TestUtils.fail();
         }
 
     }
 
-    public PrimitiveMathTest() {
-        super();
-    }
-
-    public PrimitiveMathTest(final String arg0) {
-        super(arg0);
-    }
-
+    @Test
     public void testCompareToZeros() {
 
         final double negDbl = -0.0;
         final double posInt = 0;
         final double posDbl = 0.0;
         final double negInt = -0;
 
-        PrimitiveMathTest.compare("negDbl <-> posInt", negDbl, posInt);
-        PrimitiveMathTest.compare("negDbl <-> posDbl", negDbl, posDbl);
-        PrimitiveMathTest.compare("negDbl <-> negInt", negDbl, negInt);
+        PrimitiveMathTest.compare(negDbl, posInt);
```

Comment:
Looks awful, I know - but then `compare` was not _using_ that string. We can roll that change back for now and fix `compare` instead.